### DISCUSSION TITLE
Will I be able to publish with "Individual Apple Store Connect API key"?

### DISCUSSION THREAD

**Discussion by akihiro-moriwaki** [codemagic-ci-cd/codemagic-docs#2676] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

iOS (Swift, Objective-C)

### Steps to reproduce

I think it only supports "team keys" at the moment.

https://github.com/orgs/codemagic-ci-cd/discussions/2494

### Expected results

Successful publishing with "Individual Apple Store Connect API key"

### Actual results

Will it be possible to do this with codemagic now that fastlane supports "Individual Apple Store Connect API key"?
https://github.com/fastlane/fastlane/issues/21959

**Comment by himesh-cm** [comment]
Hey @akihiro-moriwaki,
You can try it using Fastlane if it supports it. Please let us know if you face any issues during implementation.

**Comment by himesh-cm** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
